### PR TITLE
test: add unit test for updating financial transaction #215

### PR DESCRIPTION
**Pull request by sylvvester**
- test for updating financial transaction with invalid id in FinancialTransactionService
- test for updating a financial transaction with valid parameters in FinancialTransactionService

### CODE REVIEW

**Comment by mirekgab** [comment]
This should be merged after PR 223 merging.

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java [hunk 1]
```diff
@@ -242,4 +298,7 @@ private FinancialTransactionCreateDTO createFinancialTransactionCreateDTO(){
     private FinancialTransactionDTO createFinancialTransactionDTO(){
         return new FinancialTransactionDTO(ID_1L, ONE, DESCRIPTION, EXPENSE, DATE_NOW, null);
     }
+    private FinancialTransactionUpdateDTO createFinancialTransactionUpdateDTO() {
+        return new FinancialTransactionUpdateDTO(TEN,DATE_NOW, DESCRIPTION, EXPENSE,null);
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java:302]: fix code formatting

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java [hunk 2]
```diff
@@ -135,6 +135,63 @@ void testCreateFinancialTransaction_WhenDescriptionIsEmpty_ThenCreateFinancialTr
...
+    void shouldThrowExceptionWhenUpdatingFinancialTransactionWithInvalidId() {
+        //given
+        FinancialTransactionUpdateDTO updateDTO = createFinancialTransactionUpdateDTO();
+        when(financialTransactionRepository.findById(any())).thenReturn(Optional.empty());
+
+        //when & then
+        AppRuntimeException exception = assertThrows(
+                AppRuntimeException.class,
+                () -> financialTransactionService.updateFinancialTransaction(ID_1L, updateDTO)
+        );
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java:149]: You can add one empty line before block assertion code

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java [hunk 3]
```diff
@@ -135,6 +135,63 @@ void testCreateFinancialTransaction_WhenDescriptionIsEmpty_ThenCreateFinancialTr
...
+        assertEquals(ErrorCode.FT001.getBusinessStatusCode(), exception.getBusinessStatusCode());
+        assertEquals(ErrorCode.FT001.getBusinessMessage(), exception.getBusinessMessage());
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImplTest.java:151]: You can add one empty line before verifying the block code - just like you did in `shouldUpdateFinancialTransactionWhenValidParametersAreGiven()` test method.

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Request changes

**Review by dawciobiel — APPROVED** [review]
Looks good to me

**Review by Iwona007 — APPROVED** [review]

**Pull request merged**